validation/src/main/scala/hmda/validation/rules/ts/syntactical/S100.scala

```diff
@@ -1,14 +1,17 @@
 package hmda.validation.rules.ts.syntactical
 
 import hmda.model.fi.ts.TransmittalSheet
-import hmda.validation.dsl.{ CommonDsl, Result }
+import hmda.validation.dsl.Result
 
 import scala.concurrent.{ ExecutionContext, Future }
 
 /*
  Activity year must = year being processed (i.e. = 2016)
  */
-object S100 extends CommonDsl {
+object S100 {
```

Comment:
Same reason as above, the `apply` method has a different signature, taking two parameters